jni/src/main/java/com/horizen/cswnative/CswProof.java

```diff
@@ -4,13 +4,27 @@
 
 import com.horizen.certnative.WithdrawalCertificate;
 import com.horizen.librustsidechains.FieldElement;
-import com.horizen.provingsystemnative.CreateProofResult;
 import com.horizen.provingsystemnative.ProvingSystemType;
 
 public class CswProof {
+
+    private final int rangeSize;
+    private final int numCustomFields;
```

Comment:
honestly, previous structure was better as for me.
I mean, no constructor with `rangeSize` and `numCustomFields`, but mking them an args of `static` methods: setup (...) and proof(...)
PS: we never keep an instance of `CswProof` in SDK, but wrap the static methods in a java friendly way.

Reply:
I need them for creating the proof that's why I put them as members of the class. Of course, if you are more comfortable, I can just duplicate them in the static prover methods, no problem